libs/ui/tabs/helm/src/lib/hlm-tabs-paginated-list.component.ts

```diff
@@ -63,12 +63,12 @@ import { listVariants } from './hlm-tabs-list.component';
 export class HlmTabsPaginatedListComponent extends BrnTabsPaginatedListDirective {
 	@ContentChildren(BrnTabsTriggerDirective, { descendants: false })
 	_items!: QueryList<BrnTabsTriggerDirective>;
```

Comment:
Looks like this one was missed 🙂

Reply:
I'm on it!

Reply:
When changing `items` to `contentChildren(BrnTabsTriggerDirective, { descendants: false });` the tabs are not displayed when using keyboard navigation from first to the last item. I will try to find a solution and open a new pull request.

![CleanShot 2024-11-11 at 15 09 41](https://github.com/user-attachments/assets/efb79c8e-ba4a-4cd1-a804-55f2049538a6)